portfolio/src/main/webapp/script.js

```diff
@@ -258,3 +250,19 @@ function displayForm(isUserLoggedIn) {
   }
 }
 
+function loadMapMarkers(mapElement) {
+  const latitude = 33.745972;
+  const longitude = -84.413879; 
+  const zoomSize = 3;  
+  const map = new google.maps.Map(mapElement, {center: {lat: latitude, lng: longitude}, zoom: zoomSize});
```

Comment:
Creating the map inside a function called "loadMapMarkers" is a little weird. You should probably have one function that creates the map and another that adds the markers, accepting the Map object as an argument.